app/lib/server/functions/getFullUserData.js

```diff
@@ -89,6 +89,10 @@ export function getFullUserDataByIdOrUsername({ userId, filterId, filterUsername
 		fields,
 	};
 	const user = Users.findOneByIdOrUsername(filterId || filterUsername, options);
+	if (!user) {
+		return;
```

Comment:
As the function is expected to return _something_, it's nice to be a little more explicit on `return` statements:

```suggestion
		return null;
```